src/components/controls/panel-toggles.tsx

```diff
@@ -1,34 +0,0 @@
-import React from "react";
-import { useSelector } from "react-redux";
-import { useAppDispatch } from "../../hooks";
-import { useTranslation } from 'react-i18next';
-
-import Toggle from "./toggle";
-import { togglePanelDisplay } from "../../actions/panelDisplay";
-import { RootState } from "../../store";
-
-export default function PanelToggles() {
-  const dispatch = useAppDispatch();
-  const { t } = useTranslation();
-
-  const panelsAvailable = useSelector((state: RootState) => state.controls.panelsAvailable);
-  const panelsToDisplay = useSelector((state: RootState) => state.controls.panelsToDisplay);
-  const showTreeToo = useSelector((state: RootState) => state.controls.showTreeToo);
-
-  const panels = panelsAvailable.slice();
```

Comment:
> Add entropy toggle, keeping [conditions](https://github.com/nextstrain/auspice/blob/8377c4c47003468dab5b34cc02ab2d06eb952ca1/src/actions/recomputeReduxState.js#L289-L296) in mind

The intention of the code is that if the entropy panel is available for display it'll be in  `state.controls.panelsAvailable`. The conditions in that linked comment should are used to determine if "entropy" is an entry in `panelsAvailable` and shouldn't need to be re-examined.